docs/cli.rst

```diff
@@ -1,11 +1,248 @@
-The ``cci`` command line
+The ``cci`` Command Line
 ========================
 
-Basic operation
+Basic Operation
 ---------------
+To see the available commands, you can simply type ``cci``:
 
-Getting help
-------------
+.. code-block:: console
+
+    $ cci
+    Usage: cci [OPTIONS] COMMAND [ARGS]...
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    error    Get or share information about an error
+    flow     Commands for finding and running flows for a project
+    org      Commands for connecting and interacting with Salesforce orgs
+    project  Commands for interacting with project repository configurations
+    service  Commands for connecting services to the keychain
+    shell    Drop into a Python shell
+    task     Commands for finding and running tasks for a project
+    version  Print the current version of CumulusCI
+
+To get information on a specific command listed above, we can type ``cci <command>``.
```

Comment:
This paragraph and the following one switches from second person to first plural. I'd suggest sticking with the former.